docs/extensibility.md

```diff
@@ -0,0 +1,94 @@
+# Extensibility
+
+Extensibility is key for WordPress and like the rest of WordPress components, Gutenberg is hightly extensible.
+
+
+## Creating Blocks
+
+Gutenberg is about blocks and the main extensibility API of Gutenberg is the Block API. It allows you to create static blocks, dynamic blocks rendering on the server and also blocks saving data to Post Meta for more structured content.
+
+If you want to learn more about block creations, The [Blocks Tutorial](./blocks) is the best place to sart.
+
+
+## Removing Blocks
+
+### Using a black list
+
+Adding blocks is easy enough, removing them is as easy. Plugin or theme authors have the possibility to "unregister" blocks.
+
+```js
+// myplugin.js
+
+wp.blocks.unregisterBlockType( 'core/verse' );
```

Comment:
Does it mean we married `wp.blocks` forever?

Reply:
I think so :)